src/ripple/server/Port.h

```diff
@@ -58,6 +58,9 @@ struct Port
     // port in the range [0, 65535] where 0 means unlimited.
     int limit = 0;
 
+    // Websocket disconnects if send queue exceeds this limit
+    std::uint16_t ws_queue_limit = 0;
```

Comment:
I don't like duplicating the raw value in two places - at the very least, we should have a `constexpr` variable that we use, so we don't forget to update one value in the future.

More generally, I think that there's a smell coming from here...